MAUI/AIAssistView/styles.md

```diff
@@ -1041,4 +1041,52 @@ public MainPage()
 {% endhighlight %}
 {% endtabs %}
 
-![Stop responding style in .NET MAUI AI AssistView](images/styles/maui-aiassistview-styles-stopresponding.png)
\ No newline at end of file
+![Stop responding style in .NET MAUI AI AssistView](images/styles/maui-aiassistview-styles-stopresponding.png)
+
+## Text Selection styling
+
+To apply styles to highlight text for the Text Selection feature, set values to the built-in keys of the response item in the resource dictionary.
```

Comment:
To modify the highlight color of selected text, you need to update the value associated with the built-in key `SfAIAssistViewSelectionTextHighLightColor` in the resource dictionary. 

Also include the new theme keys in common themes UG page.